### PR TITLE
OSDOCS-3839: Adding Azure Active Directory to supported OIDC list

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3839

Link to docs preview:
(single page) http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3839/authentication/identity_providers/configuring-oidc-identity-provider.html#identity-provider-oidc-supported_configuring-oidc-identity-provider

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
Marking WIP until I get confirmation that we're claiming support for Azure AD in 4.11

**Review by GroceryBoyJr — COMMENTED** [review]
/LGTM

File: modules/identity-provider-oidc-supported.adoc [hunk 1]
```diff
@@ -7,12 +7,13 @@
...
 
-* Active Directory Federation Services
+* Active Directory Federation Services for Windows Server
 +
 [NOTE]
 ====
-Currently, it is not supported to use Active Directory Federation Services with {product-title} when custom claims are used.
+Currently, it is not supported to use Active Directory Federation Services for Windows Server with {product-title} when custom claims are used.
 ====
+* Azure Active Directory
```
Comment (xingxingxia) [inline modules/identity-provider-oidc-supported.adoc:16]: Same two comments apply to this PR as https://github.com/openshift/openshift-docs/pull/47921

**Pull request merged**